### PR TITLE
Feature/auth tab

### PR DESCRIPTION
**Pull request by sssoja**

### CODE REVIEW

File: dashboard-frontend/src/pages/AccountsPage/AccountList.tsx [hunk 1]
```diff
@@ -0,0 +1,184 @@
+import React, { useState } from 'react';
+import { AccountResource } from '../../types/account';
+import { arrayMerge } from '../../helpers/arrays';
+import PrimaryButton from '../../components/PrimaryButton';
+import InputSecret from './InputSecret';
+import InputToken from './InputToken';
+import useCreateAccount from './useCreateAccount';
+import useUpdateAccount from './useUpdateAccount';
+
+const staticAccountResources = [
```
Comment (Spazzy757) [inline dashboard-frontend/src/pages/AccountsPage/AccountList.tsx:10]: Im assuming these are all the account types we support?

File: dashboard-frontend/src/pages/AccountsPage/InputToken.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import React, { ChangeEventHandler } from 'react';
+import { classNames } from '../../helpers/strings';
+
+const InputToken = ({
```
Comment (Spazzy757) [inline dashboard-frontend/src/pages/AccountsPage/InputToken.tsx:4]: I noticed that we show these in plain text, which is fine for a first iteration but in general secret values like tokens and client secrets we should set the input type to password (potentially with the ability to unhide them)

**Comment by Spazzy757** [comment]
Looks good! I'll approve for now but set a label "blocked" until we get the dashboard API bit out or else this will break the frontend

File: dashboard-frontend/src/pages/AccountsPage/AccountList.tsx (hunk 1 above)
Comment (sssoja) [reply dashboard-frontend/src/pages/AccountsPage/AccountList.tsx:10]: Exactly, except test!

File: dashboard-frontend/src/pages/AccountsPage/InputToken.tsx (hunk 2 above)
Comment (sssoja) [reply dashboard-frontend/src/pages/AccountsPage/InputToken.tsx:4]: Sounds reasonable! created an issue for it https://github.com/vlab-research/vlab/issues/111

**Pull request closed without merging**